student.rb

```diff
@@ -1,9 +1,10 @@
-require_relative 'person'
-
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    classroom&.add_student(self)
   end
 
```

Comment:
- [ ] I suggest that you add a method that serves as the 'belongs-to' side (a student belongs to a classroom). Link to the article with examples: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships`
  
  ```ruby
  def classroom=(classroom)
        @classroom = classroom
        classroom.students.push(self) unless classroom.students.include?(self)
    end
  ```